terminus-ui/src/confirmation/confirmation.directive.ts

```diff
@@ -76,6 +123,17 @@ export class TsConfirmationDirective implements OnDestroy, OnInit {
   }
 
 
+  /**
+   * Dismiss the confirmation overlay on pressing escape
+   *
+   * @param event - The Keyboard Event
+   */
+  @HostListener('document:keydown.escape', ['$event'])
```

Comment:
Double check me on this.. but I think we don't need the event param here since we just want to know when this thing happened. So, 

```
@HostListener('document:keydown.escape', ['$event'])
// could be
@HostListener('document:keydown.escape')
```

Reply:
[$event] is not needed.